ignitionrobotics/ign-gazebo
# Support setting pose on links, visuals, collision via the GUI

# 🎉 New feature

## Summary

This PR allows you to edit the pose of models, links, collisions, and visuals via the component inspector. Edits have to take place while simulation is paused.

## Test it

1. `ign gazebo -v 4 shapes.sdf`
2. Apply some edits.
3. Open up another GUI: `ign gazebo -v 4 -g`
4. Step simulation in the first GUI instance. You should see changes applied in the second
5. Saving the world should also contain the pose changes.
6. Plotting pose values should also still work.

## Checklist

**Note to maintainers**: Remember to use **Squash-Merge**
